### PR TITLE
tests: add network-observe interface spread test

### PR DESCRIPTION
**Pull request by fgimenez**

### CODE REVIEW

File: tests/lib/snaps/network-observe-consumer/bin/consumer [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env python3
+
+import subprocess
+import sys
+
+def run():
+    print(subprocess.check_output("netstat -lnt", shell=True))
```
Comment (mvo5) [inline tests/lib/snaps/network-observe-consumer/bin/consumer]: You could use  `subprocess.check_call(["netstat", "-lnt"])` here (which is slightly shoter). The check_call() will simply use the existing stdout, stderr. But this is fine of course, just a few bytes longer :)

**Comment by mvo5** [comment]
Looks fine. Thank you for this branch!

**Comment by fgimenez** [comment]
@mvo5 thanks for the review! comment addressed, superuseful :)

**Comment by fgimenez** [comment]
retest this please

**Comment by chipaca** [comment]
:+1:

**Pull request merged**